src/Bicep.Core/Syntax/ParameterLiteralValueSyntax.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Bicep.Core.Parsing;
+
+namespace Bicep.Core.Syntax
+{
+    public class ParameterLiteralValueSyntax : SyntaxBase
```

Comment:
This can be deleted as we are not using it in the parser. Initially we thought parameter values can only be literal values, but on second thought it makes more sense to accept an expression as a paramter value at syntactic level (we are already calling `Expresssion()` to parse parameters values anyway). We can implement semantic checks to block the use of expressions later.